packages/reports/src/Components/new-affiliate-account-modal.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Icon, Modal, Text, Button } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import PropTypes from 'prop-types';
+import '../sass/app/_common/components/new-affiliate-account-modal.scss';
+
+const affiliateAccountOptions = [
+    { icon: 'withdrawal', title: 'Withdrawals' },
```

Comment:
Do we need to localize the titles too?

Reply:
I guess we need that, because maybe the user changed the language